app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -24,19 +27,41 @@ public ProductService(
         this.productRepository = productRepository;
     }
 
+    /**
+     * 이 어플리케이션에 있는 모든 상품목록을 리턴합니다.
+     * @return 모든 상품들.
+     */
     public List<Product> getProducts() {
         return productRepository.findAll();
     }
 
+    /**
+     * 주어진 id에 해당하는 상품을 리턴합니다.
+     * @param id 상품을 식별하는 고유 값.
+     * @return id에 해당하는 상품.
+     * @throws ProductNotFoundException 상품을 찾을 수 없을 경우 발생.
```

Comment:
`@throws`에 대해서는 `~한 경우`로 명시해 보세요. 
```suggestion
     * @throws ProductNotFoundException 상품을 찾을 수 없을 경우
```